_articles/send/send-cli.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: article
+title: Send from CLI
+categories: [send]
+featured: true
+popular: false
+tags: []
+order: 06
+---
+
+Bitwarden Send is available as a set of fully-featured CLI commands. This article documents the breadth of `bw send` commands, however Send **is not a separate tool** from the Bitwarden Command-line Interface (CLI). Therefore, many of the commands, options, and concepts in the [CLI documentation]({% link _articles/miscellaneous/cli.md %}) are relevant here.
+
+{% image /send/send-cli.png Send's --help text %}
+
+## send
+
+The `send` command is the master command used to access all Send-related subcommands:
+
+```
+bw send [options] [command] <data>
+```
+
+The `send` command can **also** can also be used as a shortcut to quickly `create` a Send, for example:
```

Comment:
@fschillingeriv 
> The `send` command can **also** can also be used as a shortcut to quickly `create` a Send, for example:

The `send` command can **also** be used as a shortcut to quickly `create` a Send, for example: